lib/common/Makefile.am

```diff
@@ -110,8 +105,5 @@ libcrmcommon_test_la_CFLAGS	= $(libcrmcommon_la_CFLAGS)
 libcrmcommon_test_la_LIBADD	= $(libcrmcommon_la_LIBADD)
 nodist_libcrmcommon_test_la_SOURCES = $(nodist_libcrmcommon_la_SOURCES)
 
-md5.c: ../gnu/md5.c
-	cp "$<" "$@"
-
```

Comment:
I guess we also need to move md5.c to the "Formerly built" section

Reply:
Oh, and I should drop it from MOSTLYCLEANFILES in Makefile.am as well.